js/src/get-started-page/get-started-card/index.scss

```diff
@@ -1,41 +1,60 @@
-.woocommerce-marketing-google-get-started-card {
-	.components-flex {
-		flex-direction: column;
-		align-items: flex-end;
+@import "../_shared";
```

Comment:
🚧 
Although **_shared.scss** was not created in this PR, it's worth mentioning that Sass `@import` makes all scss variables in the imported file globally accessible across all scss files. Variables pollution globally further cause the collisional variables to overwrite each other in the order they are referenced, making debugging and maintenance difficult.

Although Sass `@use` can keep scss variables within the scope of individual scss files, the [current webpack config](https://github.com/woocommerce/google-listings-and-ads/blob/587422aaa78b275c173009c14ea87a5dc9498816/webpack.config.js#L127-L131) inserts other `at-rules` at the first line of every scss file, making it impossible to use `@use` for individual scss files. It's recommended to add a `gla-` prefix to variables and move all shared variables to **js/src/css/_variables.scss** for better maintainability.

Ref: https://sass-lang.com/documentation/at-rules/import

Reply:
:til: Thanks for spotting this @eason9487 I didn't know that one neither and missed that one in my review.

Reply:
Thanks for catching this @eason9487 as well. I ended up adding the scss file under `js/src/css/abstracts` and named it `_gla_variables.scss` and also added `@import "_gla_variables"` in `webpack.config.js` as this was the only way I could make it work without extra `@import` from other scss files. The commit for this is 302d769.